src/badge/lib/badge.scss

```diff
@@ -0,0 +1,34 @@
+$_tbds-badge-font-size: 0.8em !default;
+$_tbds-badge-background-color: #f5f5f5 !default;
```

Comment:
This is feeling a bit light to me, what do you think about darkening it up some?

Reply:
Sure. I was pulling from Constable, but I'll see if I can snag a better gray from the brand guidelines.

Reply:
If you’d rather ignore this for now, that’s fine too, since it looks like we’ll be fleshing out more of the color system soon.

Reply:
Yeah, I think that's probably wisest.